shared/js/pageStore.js

```diff
@@ -65,11 +64,9 @@ loop.store.PageStore = function() {
      */
     addedPage(actionData) {
```

Comment:
having `addPage` and `addedPage` is a little bit confusing, I'd rather go for `addPage` for the intention of adding a new one, and `pageAdded` for the consequences after adding it, but I understand it's not the scope of this PR, so at the moment, enough with a comment like `// XXX rename for readability`